subt/rosbag2log.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/python
+"""
+  Statistics on ROSbag data
+  usage:
+       ./rosbag.py <input file>
+"""
```

Comment:
I am not reviewing this file since I don't want to be reimplementing rosbag.

Reply:
OK